setup_scripts/moab_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ 
+#MOAB Installation
+INSTALL_ROOT=$HOME
+cd $INSTALL_ROOT
+mkdir MOAB
+cd MOAB
+mkdir bld
+mkdir install
+git clone https://bitbucket.org/fathomteam/moab
+cd moab
+git checkout Version5.0
+autoreconf -fi
+cd ../bld
+../moab/configure --prefix=$INSTALL_ROOT/MOAB/install --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial --enable-shared
+make -j4
+make check
+make install
+printf 'export PATH=$PATH:$INSTALL_ROOT/MOAB/install/bin'>>$HOME/.bashrc
```

Comment:
Might be better to print this out as guidance for the user to make this change, rather than making it automatically.

Also probably useful to prepend the new path since you want to look there first.